src/python/pants/pantsd/process_manager.py

```diff
@@ -86,15 +49,9 @@ class Timeout(Exception):
     INFO_INTERVAL_SEC = 5
     WAIT_INTERVAL_SEC = 0.1
 
-    def __init__(self, metadata_base_dir=None):
-        """
-        :param str metadata_base_dir: The base directory for process metadata.
-        """
+    def __init__(self, metadata_base_dir: str) -> None:
```

Comment:
This was always configured in production. It allows us to remove `Subprocess`, which I'm not sure even worked still.